boards/DCM/Src/main.c

```diff
@@ -183,15 +186,20 @@ int main(void)
 
     buzzer = App_Buzzer_Create(Io_Buzzer_TurnOn, Io_Buzzer_TurnOff);
 
+    imu = App_Imu_Create(
+            Io_LSM6DS33_ExtiCallback, Io_LSM6DS33_GetAccelerationX,
```

Comment:
The IMU object does not need the callback function since `Io_LSM6DS33_ExtiCallback` will be called inside HAL_GPIO_EXTI_Callback.

Reply:
I removed it but added a get_exit_code function so that the function which broadcasts Imu data can know when there's an error and broadcast it